tesseract_process_managers/include/tesseract_process_managers/task_generators/iterative_spline_parameterization_task_generator.h

```diff
@@ -76,7 +76,8 @@ class IterativeSplineParameterizationTaskInfo : public TaskInfo
 }  // namespace tesseract_planning
 
 #ifdef SWIG
-%tesseract_command_language_add_profile_type(IterativeSplineParameterizationProfile);
+//TODO: Where did IterativeSplineParameterizationProfile go?
```

Comment:
They are now located in [this](https://github.com/tesseract-robotics/tesseract_planning/tree/master/tesseract_process_managers/include/tesseract_process_managers/task_profiles) folder.